DEVELOPERS.md

```diff
@@ -483,4 +483,4 @@ You can see an example of a well-defined example [in the `ngRepeat` documentatio
 [karma-browserstack]: https://github.com/karma-runner/karma-browserstack-launcher
 [karma-saucelabs]: https://github.com/karma-runner/karma-sauce-launcher
 [unit-testing]: https://docs.angularjs.org/guide/unit-testing
-[yarn-install]: https://yarnpkg.com/en/docs/install
\ No newline at end of file
+[yarn-install]: https://yarnpkg.com/en/docs/install
```

Comment:
what changed here?

Reply:
A new line was added to the end of the file.